app/assets/javascripts/browse-columns.js

```diff
@@ -337,7 +337,10 @@
       }
     },
     updateBreadcrumbs: function(state) {
-      var $newBreadcrumbs = $(state.sectionData.breadcrumbs);
+      // Calling `.last()` is necessary because the breadcrumbs component has 2
```

Comment:
Will this still work if Slimmer moves the `<script>` to the bottom of the page?

Reply:
Yeah, `state.sectionData` comes directly from the [json endpoint](https://github.com/alphagov/collections/blob/73ad7c021500b31881fbb0a6a51fba5607c303bb/app/controllers/browse_controller.rb#L22-L27) so slimmer doesn't interfere.